ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -395,6 +396,22 @@ local_users:
     ssh_key: "< ssh_key_string >"
 ```
 
+#### User Roles
+
+```yaml
+user_roles:
```

Comment:
1. Make sure to keep the model close to EOS. Ex. let's call the main key `roles`.
2. Going forward we will try to move to list of dicts instead of dicts with wildcard keys. This is to be able to define a schema and input validation as well as code efficiency.
3. I would suggest that we take more of the EOS syntax into the data model. My suggestion:
```yaml
roles:
  - name: < role_name >
    sequence_numbers:
      - sequence: < sequence_number >
        action: < permit | deny >
        mode: < "config" | "config-all" | "exec" | "<mode>" >
        command: < command as string >
```

Reply:
adjusted the data model